CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 **Internal:**
 
 - Support downstream Xbox SDK specifying networking initialization mechanism. ([#1359](https://github.com/getsentry/sentry-native/pull/1359))
+- Implement logging enable/disable feature, with crash handler options ([#1371](https://github.com/getsentry/sentry-native/pull/1371))
```

Comment:
This adds an externally visible API/option so should be under Features, not Internal. Also, consider rewording, for example:
```suggestion
- Add a configuration to disable logging in crash handlers - `sentry_options_set_logger_is_signal_safe()` ([#1371](https://github.com/getsentry/sentry-native/pull/1371))
```